### PR TITLE
Add documentation for sharing and roles

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

File: neuro-cli/src/neuro_cli/topics.py [hunk 1]
```diff
@@ -458,3 +458,86 @@ async def secrets() -> None:
...
+
+
+@topics.command()
+async def sharing() -> None:
+    """
+    Using sharing functionality
+    ===========================
+
+    Understanding permissions
+    =========================
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/topics.py:470]: I expect header level 2 (`----`) here and below

File: neuro-cli/src/neuro_cli/topics.py [hunk 2]
```diff
@@ -458,3 +458,86 @@ async def secrets() -> None:
...
+    * deny - no access
+    * list - permits listing entities, but not looking at their details
+    * read - read-only access to entity
+    * write - write-only access (including deletion) to entity
+    * manage - allows modification of entity's permissions
+
+    Please note permissions are inclusive: write permission implies read and manage
+    includes read and write, and so on.
+
+    Permissions can be granted via "`neuro acl grant`" or "`neuro share`" and
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/topics.py:481]: quotas are redundant if you have backticks already.

**Comment by asvetlov** [comment]
@serhiy-storchaka please review

File: neuro-cli/src/neuro_cli/topics.py (hunk 2 above)
Comment (anayden) [reply neuro-cli/src/neuro_cli/topics.py:481]: Terminal output without quotes is difficult to read in my opinion. Backticks doesn't change anything visually (at least in my macOS terminal).

Comment (asvetlov) [reply neuro-cli/src/neuro_cli/topics.py:481]: I believe we should change markdown styling, not the original text.

If you want write the same markdown for web, would you use quotas or backticks only? I think the second.

Comment (anayden) [reply neuro-cli/src/neuro_cli/topics.py:481]: I don't think click's `HelpFormatter` doesn't expose anything that allows formatting of terminal output. I will remove quotes and keep backticks.

File: neuro-cli/src/neuro_cli/topics.py [hunk 3]
```diff
@@ -376,14 +376,14 @@ async def aliases() -> None:
...
-    e.g. `exec = "du {human_readable} {max_depth} {file}"`.
+    e.g. `exec = "du {human_readable} {max_depth} {file}`.
     Substitution is a variable name to expand enclosed in figure brackets,
     e.g. `{file}`.
 
     It is expanded with an option or positional argument specified
     by `args` or `options`.  The substitution name is automatically lowercased,
     minus (`-`) is replaced with underscore (`_`).
-    E.g. `args = "ARG-NAME"` matches to `{arg_name}`.
+    E.g. `args = "ARG-NAME` matches to `{arg_name}`.
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/topics.py:386]: Something wrong here

File: neuro-cli/src/neuro_cli/topics.py [hunk 4]
```diff
@@ -367,7 +367,7 @@ async def aliases() -> None:
     finish, and, in turn, returns the exit code to the outer caller.
 
     The parameter may specify and executable file along with some options,
-    e.g. `exec = "du --human-readable"` enforces human-readable mode
+    e.g. `exec = "du --human-readable` enforces human-readable mode
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/topics.py:370]: Looks broken

File: neuro-cli/src/neuro_cli/topics.py [hunk 5]
```diff
@@ -376,14 +376,14 @@ async def aliases() -> None:
     ------------
 
     In **pattern mode** the system command is used along with **substitutions**,
-    e.g. `exec = "du {human_readable} {max_depth} {file}"`.
+    e.g. `exec = "du {human_readable} {max_depth} {file}`.
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/topics.py:379]: Revert please

File: neuro-cli/src/neuro_cli/topics.py [hunk 6]
```diff
@@ -405,15 +405,15 @@ async def aliases() -> None:
     In **simplified mode** the `exec` value does not contain any **substitutions**.
     In this case all parsed `options` and `args` are appended
     to executed command automatically if provided,
-    e.g. `exec = "du"` is expanded to
-    `exec = "du {human_readable} {max_depth} {file}"`
+    e.g. `exec = "du` is expanded to
+    `exec = "du {human_readable} {max_depth} {file}`
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/topics.py:409]: Revert please back

**Review by asvetlov — COMMENTED** [review]
Please revert unrelated chages and merge

File: neuro-cli/src/neuro_cli/topics.py (hunk 3 above)
Comment (anayden) [reply neuro-cli/src/neuro_cli/topics.py:386]: Sorry, I was 100% sure I only committed related chunks and not these ones. My apologies for not double-checking this fact in the PR.

**Review by asvetlov — APPROVED** [review]
Thanks!

**Pull request merged**